src/solvers/dgsem_tree/dg_2d_parabolic.jl

```diff
@@ -32,6 +32,17 @@ function rhs_parabolic!(du, u, t, mesh::TreeMesh{2}, equations_parabolic::Abstra
 
   # The remainder of this function is essentially a regular rhs! for parabolic equations (i.e., it
   # computes the divergence of the viscous fluxes)
+  #
+  # OBS! In `calc_viscous_fluxes!`, the viscous flux values at the volume nodes of each element have
```

Comment:
Minor question, what does `OBS!` mean here?

Reply:
AFAIK, that's @andrewwinters5000 way of escalating a "note" to a "beware" kind of comment 😬 If I misused this, however, I am happy to change it back.

Reply:
Whoops, yes. That is my funny way of putting a note that is also a warning for a user. It is the Swedish version of "Achtung!" it literally means "Observera att!" We can change this to something else if you want 😬

Reply:
Haha! I like it. I assumed it meant "observation", which is similar enough. I'm good with leaving it.